GVRf/Sample/gvrurlmodelloader/src/org/gearvrf/urlmodelloader/URLModelLoaderViewManager.java

```diff
@@ -0,0 +1,203 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gearvrf.urlmodelloader;
+
+import java.io.BufferedInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.concurrent.ExecutionException;
+
+import org.gearvrf.GVRCameraRig;
+import org.gearvrf.GVRContext;
+import org.gearvrf.GVRScene;
+import org.gearvrf.GVRSceneObject;
+import org.gearvrf.GVRScript;
+import org.gearvrf.utility.Log;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import android.app.ProgressDialog;
+import android.graphics.Color;
+import android.os.AsyncTask;
+import android.os.Environment;
+
+public class URLModelLoaderViewManager extends GVRScript {
+
+    JSONObject jsonobject;
+    JSONArray jsonarray;
+    ProgressDialog mProgressDialog;
+    RemoteFileAttributes modelUtils = new RemoteFileAttributes();
+    private GVRScene scene;
+
+    @Override
+    public void onInit(GVRContext gvrContext) throws InterruptedException, ExecutionException, IOException {
+
+        scene = gvrContext.getNextMainScene();
+        scene.setFrustumCulling(true);
+
+        // set background color
+        GVRCameraRig mainCameraRig = scene.getMainCameraRig();
+        mainCameraRig.getLeftCamera().setBackgroundColor(Color.BLACK);
+        mainCameraRig.getRightCamera().setBackgroundColor(Color.BLACK);
+        mainCameraRig.getTransform().setPosition(0.0f, 0.0f, 0.0f);
+
+        String jsonFileURL = "https://raw.githubusercontent.com/lokeshguddu/lokeshguddu.github.io/master/remote_model_urls.json";
```

Comment:
The url probably needs to change to the official GearVRf-Demos repo. Also please move the sample to that repo (we will remove all the samples from this repo soon). The samples repo is https://github.com/gearvrf/GearVRf-Demos.